packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -327,11 +330,12 @@ describe('Checkout', () => {
                     localVue,
                     propsData
                 });
+                await flushPromises();
```

Comment:
Just curious, but why did we need to add this `flushPromises` in these tests? 🙂

Reply:
The test framework seems to return from mount to the test after first await. I now added another call so the test doesn't wait for this call to complete unless the flushPromises is added